docs/libcurl/curl_url_get.3

```diff
@@ -84,7 +84,14 @@ If the host name is a numeric IPv6 address, this field might also be set.
 .IP CURLUPART_PORT
 Port cannot be URL decoded on get.
 .IP CURLUPART_PATH
+\fIpart\fP will always start with a `/' even with a zero-length path.
```

Comment:
I think I understand what you mean, but it sounds funny. It can't be zero.-length **and** have a slash in it! :grimacing:

Reply:
Oh, IIRC I tried that URI "http://example.com?foo" and the path was still being set to "/".

Reply:
Not quite sure if this is considered zero-length. Haven't read RFC 7230 & 3986 for a while ;)

Reply:
Yes, a zero length *path input* will make a `/` path *output*. My point was that it isn't clear in the text that you are talking about the input vs output.

Reply:
Oh how about
> part will always start with a `/' even with a zero-length path supplied.

Reply:
If we're talking about the case when no path is provided, it could be made even more explicit:

> **part** will be `/` even if no path is supplied in the URL.

Reply:
Agreed, this is more understandable for people without knowledge about the specification